docs/app/documentation/component-docs/localization-editor/localization-editor-docs.component.html

```diff
@@ -0,0 +1,56 @@
+<h2>Localization Editor</h2>
+
+<description>
+    Basic usage of Localization Editor, to add some text on add-on there is <code>[label]</code> input.
+</description>
+<component-example [name]="'ex1'">
+  <p>
+    <fd-localization-editor-example></fd-localization-editor-example>
+  </p>
+</component-example>
+<code-example [code]="localizationHtml" [language]="'HTML'"></code-example>
+
+<separator></separator>
+
+<h2>Forms in Localization Editor</h2>
+<description>
+    Forms usage inside localization editor.
```

Comment:
You don't really use the forms inside the editor, you use the editor in a form. Maybe 'The localization editor component is compatible with Angular forms.' or something similar